File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.swift
class Solution {
    func totalMoney(_ n: Int) -> Int {
        var result = 0
        
        for m in 1...n {
            let div = m / 7
            let mod = m % 7
            
            result += div
            
            if mod == 0 {
                result += 6
            } else {
                result += mod
            }
        }
        
        return result
    }
}

File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.py
class Solution:
    def totalMoney(self, n: int) -> int:
        result = 0
        
        for m in range(1, n+1):
            d, r = divmod(m, 7)
            result += d
            
            if r == 0:
                result += 6
            else:
                result += r
        
        return result